### PR TITLE
What is devops

### PR DESCRIPTION
**Pull request by prwolfe**
Mark,

You had initially found this article and asked me to read it. I think it is appropriate for a curated content so I did a write-up. Take a look and let me know if we should get this into the process.

Thanks!

Paul

### CODE REVIEW

File: CuratedContent/WhatIsDevOps.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# What is DevOps?
+
+## Mueller, Ernest, 2010, https://theagileadmin.com/what-is-devops
+
+DevOps has become a catch-all term in agile organizations, but defiing what it *is* turns out not to be simple.  This blog post attempts to sort through the various ideas surrounding DevOps and bring some clarity to what is, at heart, a large and nuanced concept.
```
Comment (markcmiller86) [inline CuratedContent/WhatIsDevOps.md]: Oh, I see, when you write "This blog post..." you are referring to the article you have reviewed and not to *this* artilcle. Hmm...maybe just adjust to read "Mueller's blog post..."

File: CuratedContent/WhatIsDevOps.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+> DevOps means a lot of different things to different people because the discussion around it covers a lot of ground.
+
+Mueller does a nice job of discussing many of the complexities and trade-offs in all this ground including a bit of history lesson, but two items stood out the most to me.
+
+1. The discussion focuses on web apps - a common issue for HPC practioners. Don't let that stop you from understanding and applying these concepts.
```
Comment (markcmiller86) [inline CuratedContent/WhatIsDevOps.md]: I *love* the use of *practitioners* here ;)

Hmm...what is a common issue thought...that most DevOps literature is steeped in the web app domain or that web apps are a domain that HPC practitioners commonly struggle with?

Maybe revise to "Although the discussion focuses on web apps, HPC practitioners will still benefit from many of the concepts."

**Comment by markcmiller86** [comment]
> Regarding our treatment of that content here, I was actually hoping for a _What Is DevOps?_ contribution here (I'll elaborate in a follow-up comment).

What I mean by this is that we write a longer *What Is* article that describes what DevOps is for BSSw visitors (e.g. HPC/CSE practitioners) with link(s) to articles such as the one referenced here. I think that would be very useful to our community to have a short, quick summary, steeped in HPC/CSE-speak, addressing DevOps concepts most relevent to HPC/CSE community as a quick primer.

I know that takes a lot more work than a curated content contribution. So, no worries if you don't have time for that. I don't think I do either, at least no any time soon.

File: CuratedContent/WhatIsDevOps.md (hunk 1 above)
Comment (prwolfe) [reply CuratedContent/WhatIsDevOps.md]: Nice idea @markcmiller86 - Will do

File: CuratedContent/WhatIsDevOps.md (hunk 2 above)
Comment (prwolfe) [reply CuratedContent/WhatIsDevOps.md]: My thought was that most DevOps literature is focused on web applications not that HPC people had understanding. I like your re-write, thanks!

**Comment by prwolfe** [comment]
> What I mean by this is that we write a longer _What Is_ article that describes what DevOps is for BSSw visitors (e.g. HPC/CSE practitioners) with link(s) to articles such as the one referenced here.

@markcmiller86 - this could make sense as well. I will start on this unless the larger group shoots it down. Obviously it won't show up today ;-<

**Comment by markcmiller86** [comment]
@prwolfe are you waiting on anything from me at this point?

Also, just a curiosity...you are working from a fork even though you have write access to the main repo. Any special reason?

**Comment by prwolfe** [comment]
> @prwolfe are you waiting on anything from me at this point?

No - just occupied.

> 
> Also, just a curiosity...you are working from a fork even though you have write access to the main repo. Any special reason?

Habits - I have done it this way on so many projects for so many years....
Several projects will not give anyone outside a small group write access of any sort, so you just get into the habit eventually.

**Comment by pagrubel** [comment]
Okay so I need to be honest here I am a newbie to the subject. I've heard the term but really didn't know what it means. I think his definition looks good in the article but isn't anywhere in your review. Pointing out the things that jumped out to you is good. I like the idea of having a What Is and it should be a pre requesite to this. In the mean time, would it be okay to quote his definition on your review?  "DevOps is the practice of operations and development engineers participating together in the entire service lifecycle, from design through the development process to production support. "

**Comment by rinkug** [comment]
Already published in october end

**Comment by rinkug** [comment]
Published: https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/blob/master/CuratedContent/WhatIsDevOps.md

**Pull request merged**